src/main/java/org/opensearch/ad/settings/AnomalyDetectorSettings.java

```diff
@@ -210,7 +210,17 @@ private AnomalyDetectorSettings() {}
 
     public static final int NUM_MIN_SAMPLES = 128;
 
-    public static final double DESIRED_MODEL_SIZE_PERCENTAGE = 0.0002;
+    // The threshold for splitting RCF models in single-stream detectors.
+    // The smallest machine in the Amazon managed service has 1GB heap.
+    // With the setting, the desired model size there is of 2 MB.
+    // By default, we can have at most 5 features. Since the default shingle size
+    // is 8, we have at most 40 dimensions in RCF. In our current RCF setting,
+    // 30 trees, and bounding box cache ratio 0, 40 dimensions use 449KB.
+    // Users can increase the number of features to 10 and shingle size to 60,
+    // 30 trees, bounding box cache ratio 0, 600 dimensions use 1.8 MB.
+    // Since these sizes are smaller than the threshold 2 MB, we won't split models
+    // even in the smallest machine.
+    public static final double DESIRED_MODEL_SIZE_PERCENTAGE = 0.002;
```

Comment:
Much needed. One way to view this is were more cautious when the models were 10x in size. Given that the models are smaller and are unlikely to grow fast, this makes sense. This is only 0.2 %.